action-server/tests/actions/test_assessment_form.py

```diff
@@ -347,6 +347,30 @@ def _test_mild_symptoms_cough(self, fever: bool):
             ["utter_mild_symptoms_self_isolate", "utter_ask_lives_alone"]
         )
 
+    def test_mild_symptoms_no_cough(self):
```

Comment:
Just to fit with the reste of the naming I think this should be 'test_fever_mild_symptoms_no_cough'